src/helpers/validations/userValidator.js

```diff
@@ -0,0 +1,60 @@
+import { check, validationResult } from 'express-validator/check';
```

Comment:
#### Critical
I believe you were told not to use `express-validator`. Why use it here??

Reply:
Serious, I will take care of that and repeat validations, thanks